lua/swamp/misc/sh_titles.lua

```diff
@@ -160,6 +160,69 @@ AddTitle("kleinertp", {
     {1, "Test Subject", 10000}
 }, "Be subjected to one of Dr. Isaac Kleiner's teleportation experiments", "s_kleinertp")
 
+AddTitle("buttonsearch", {
+    { 1, "Button Rookie", 0},
+    { 10, "Button Hunter", 0},
+    { 100, "Button Master", 0}
+}, "Find and press %s buttons.", "s_magicbutton")
+
+AddTitle("snowballhit", {
+    {100, "Frosty The Snowman"}
+}, "Hit %s active players with a snowball.", "s_snowballhit")
+
+AddTitle("drinkenergy", {
+    {100, "Boomer"},
+    {500, "Quake Was A Good Game"},
+    {1000, "Quake Champion"}
+}, "Drink your boomer juice %s times", "s_drinkenergy")
+
+AddTitle("bountykill", {
+    {10, "Bounty Hunter", 0},
+    {100, "The Cleaner", 0},
+    {1000, "Hitman", 0}
+}, "Collect %s bounties.", "s_bounties")
```

Comment:
s_bountykill

use a partnerset so the bounties are unique

Reply:
That is a good point, I didn't consider it for unique kills to make it more difficult to acquire. Done.